api/maven-api-core/src/main/java/org/apache/maven/api/package-info.java

```diff
@@ -30,7 +45,8 @@
  * They are created when <dfn>resolving</dfn> an {@code ArtifactCoordinates}. Resolving is the process
  * that selects a particular version and downloads the artifact in the local repository.
  * There are two sub-interfaces, {@link org.apache.maven.api.DownloadedArtifact} which is used when
```

Comment:
I think there are four of these now. You don't need to list them all here, but let's not say explicitly how many there are.